test/lib.fetching.content-services.facebook-content-service.test.js

```diff
@@ -72,35 +72,29 @@ describe('Facebook content service', function() {
 
   it('should get new comments from old posts', function(done) {
 
-    // Fetch first batch
-    var service = stubWithFixture('facebook-1.json');
-    service.fetch({maxCount: 50}, function(){});
-
-    // Stub for second batch
-    stubWithFixture('facebook-2.json', service);
-
+    var service = stubWithFixture('facebook-2.json');
```

Comment:
If I remember correctly I think I found fixtures 1 and 2 redundant so I skipped over the first one in hopes of making a different test later on. Later on is now here, so I'll switch up that fixture and add a few more tests.